libraries/BLE/src/BLEEddystoneTLM.cpp

```diff
@@ -46,10 +44,14 @@ uint16_t BLEEddystoneTLM::getVolt() {
 	return ENDIAN_CHANGE_U16(m_eddystoneData.volt);
 } // getVolt
 
-float BLEEddystoneTLM::getTemp() {
-	return ENDIAN_CHANGE_U16(m_eddystoneData.temp) / 256.0f;
+uint16_t BLEEddystoneTLM::getTemp() {
+	return ENDIAN_CHANGE_U16(m_eddystoneData.temp);
 } // getTemp
 
+float BLEEddystoneTLM::getFloatTemp() {
+  return EDDYSTONE_TEMP_U16_TO_FLOAT(ENDIAN_CHANGE_U16(m_eddystoneData.temp));
+} // getFloatTemp
+
```

Comment:
As said, this is a breaking change. I think it should stay the same:
`float BLEEddystoneTLM::getTemp()`

The original code works correctly.

Reply:
`m_eddystoneData.temp` is in Bigendian 8.8 fixed point (16 bits) and `EDDYSTONE_TEMP_U16_TO_FLOAT()` already does all the job.

No need to use `ENDIAN_CHANGE_U16()` here.

Please eliminate this method because it causes a breaking change to the API.